Resources/views/app/base.html.twig

```diff
@@ -22,7 +29,13 @@
     {{ include('partials/navbar_user.html.twig') }}
 {% endblock %}
 
-{% block title %}AAK-tid – {{ block('page_title') }}{% endblock %}
+{% block title %}
+    {% if meta_title is not empty %}
+        {{ meta_title|raw }}
```

Comment:
You probably don't want to use [`raw`](https://twig.symfony.com/doc/3.x/filters/raw.html) here. It will let tags, say, in `meta_title` pass through without escaping, and we don't want that. It's an edge case, but still …